src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Commons.cs

```diff
@@ -339,6 +349,10 @@ internal static Exception CreateException(NNStreamerError err, string msg)
                     exp = new IndexOutOfRangeException(msg);
                     break;
 
+                case NNStreamerError.OutOfMemory:
+                    exp = new OutOfMemoryException(msg);
```

Comment:
We do not throw OutOfMemoryException on F/W code

❌ DO NOT explicitly throw OutOfMemoryException. This exception is to be thrown only by the CLR infrastructure.
https://docs.microsoft.com/ko-kr/dotnet/standard/design-guidelines/using-standard-exception-types#outofmemoryexception